converters.js

```diff
@@ -209,11 +209,48 @@ getProvisionedAddressByDid.createJsObject = function (xml, did) {
         });
 };
 
+var getProvisionedAddressHistoryByDid = {};
+getProvisionedAddressHistoryByDid.createJsObject = function (xml, did) {
+    return parseXml(xml)
+        .then(result => {
+            var addresses = _.map(result['ns2:getProvisionedLocationHistoryByURIResponse'].ProvisionedLocations, locations => {
+                return {
```

Comment:
@benjcallaghan I'd recommend reusing this logic from the other, similar function, if possible.